pythran/pythonic/include/numpy/reduce.hpp

```diff
@@ -19,29 +19,6 @@ namespace pythonic
   namespace numpy
   {
 
-    template <class Op, size_t N, bool vector_form>
-    struct _reduce;
-
-    template <class Op, bool vector_form>
-    struct _reduce<Op, 1, vector_form> {
-      template <class E, class F>
-      F operator()(E e, F acc);
-    };
-
-#ifdef USE_BOOST_SIMD
-    template <class Op>
-    struct _reduce<Op, 1, true> {
-      template <class E, class F>
-      F operator()(E e, F acc);
-    };
-#endif
-
-    template <class Op, size_t N, bool vector_form>
-    struct _reduce {
-      template <class E, class F>
-      F operator()(E e, F acc);
-    };
-
```

Comment:
Nice. For others classes, I moved this part in a "details" namespace. (Only available in the "src" file except for types)